jmxtrans-output/jmxtrans-output-influxdb/src/main/java/com/googlecode/jmxtrans/model/output/InfluxDbWriter.java

```diff
@@ -156,8 +156,8 @@ public void doWrite(Server server, Query query, ImmutableList<Result> results) t
 
 			// send the point if filteredValues isn't empty
 			if (!filteredValues.isEmpty()) {
-				filteredValues.put("_jmx_port", Integer.parseInt(server.getPort()));
```

Comment:
This is a change of behaviour. I'd much prefer if it was controlled by a flag so that the current behaviour is preserved by default and the new behaviour can be enabled for users who want it.

If you think that the current behaviour does not make sense and that `_jmx_port` should only be published as a tag, than add a warn log message so that users can update their configuration.